bin/node/cli/src/command.rs

```diff
@@ -32,6 +32,9 @@ use sp_keyring::Sr25519Keyring;
 
 use std::sync::Arc;
 
+#[cfg(feature = "try-runtime")]
+use try_runtime_cli::block_building_info::timestamp_with_aura_info;
```

Comment:
I personally always prefer using the full path where needed, or doing the import more locally, rather than feature gated imports, but up to you.